### PR TITLE
Update cov pools How it Works page

### PR DESCRIPTION
**Pull request by michalsmiarowski**
Updates coverage pools How it Works page.

Also adds `react-accessible-accordion` library.

### CODE REVIEW

File: solidity-v1/dashboard/src/components/AccordionItemHeadingWithPlusMinusSign.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+} from "react-accessible-accordion"
+
+const AccordionItemHeadingWithPlusMinusSign = ({
+  children,
+  className = "",
+  ...accordionItemButtonProps
+}) => {
+  return (
+    <AccordionItemHeading
+      className={`accordion__heading--with-plus-minus-sign ${className}`}
```
Comment (r-czajkowski) [inline solidity-v1/dashboard/src/components/AccordionItemHeadingWithPlusMinusSign.jsx:15]: Based on the BEM docs we should `accordion__heading accordion__heading--with-plus-minus-sign`.

**Review by r-czajkowski — CHANGES_REQUESTED** [review]

File: solidity-v1/dashboard/src/components/AccordionItemHeadingWithPlusMinusSign.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    >
+      <AccordionItemButton {...accordionItemButtonProps}>
+        <h3>{children}</h3>
```
[the remark this answers is not in the document]
Comment (michalsmiarowski) [reply solidity-v1/dashboard/src/components/AccordionItemHeadingWithPlusMinusSign.jsx]: Fixed in c5f5ddaf3c9dc0065109954e554d8f74acd924ca

[the remark this answers is not in the document]
Comment (michalsmiarowski) [reply solidity-v1/dashboard/src/components/AccordionItemHeadingWithPlusMinusSign.jsx]: Fixed in c5f5ddaf3c9dc0065109954e554d8f74acd924ca

File: solidity-v1/dashboard/src/components/AccordionItemHeadingWithPlusMinusSign.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        <div>
+          <AccordionItemState>
+            {({ expanded }) => (expanded ? "-" : "+")}
```
[the remark this answers is not in the document]
Comment (michalsmiarowski) [reply solidity-v1/dashboard/src/components/AccordionItemHeadingWithPlusMinusSign.jsx]: Fixed in 07a944fded72d7fa23132196610a237f62fc82c4

File: solidity-v1/dashboard/src/components/AccordionItemHeadingWithPlusMinusSign.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (michalsmiarowski) [reply solidity-v1/dashboard/src/components/AccordionItemHeadingWithPlusMinusSign.jsx]: I agree!

FIxed in 193094247fc2a5118babe47df0ad10a90a462bce

File: solidity-v1/dashboard/src/pages/coverage-pools/HowItWorksPage.jsx [hunk 4]
```diff
@@ -175,48 +173,127 @@ const HowItWorksPage = () => {
...
+              </AccordionItemPanel>
+            </AccordionItem>
 
-        <section className="tile">
-          <h3 className="mb-1">covKEEP</h3>
-          <List items={about}>
-            <List.Title className="h5 text-violet-80">About</List.Title>
-            <List.Content className="bullets bullets--violet-80 text-grey-60" />
-          </List>
+            <AccordionItem>
```
[the remark this answers is not in the document]
Comment (michalsmiarowski) [reply solidity-v1/dashboard/src/pages/coverage-pools/HowItWorksPage.jsx:285]: Fixed in 900a766ebe84311fc70b79ca44d61dc6ad3370f0

File: solidity-v1/dashboard/src/components/AccordionItemHeadingWithPlusMinusSign.jsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import {
+  AccordionItemHeading,
+  AccordionItemButton,
+  AccordionItemState,
+} from "react-accessible-accordion"
+
+const AccordionItemHeadingWithPlusMinusSign = ({
```
[the remark this answers is not in the document]
Comment (michalsmiarowski) [reply solidity-v1/dashboard/src/components/AccordionItemHeadingWithPlusMinusSign.jsx]: I changed the name to `AccordionDefaultItemHeading` - please see bde8d300a2807cd1c72e200e66ad19349101a3e1

File: solidity-v1/dashboard/src/components/AccordionItemHeadingWithPlusMinusSign.jsx (hunk 1 above)
Comment (michalsmiarowski) [reply solidity-v1/dashboard/src/components/AccordionItemHeadingWithPlusMinusSign.jsx]: Fixed in b5e3837efee85365e215e2b9166c0c7f59ab563c

**Review by r-czajkowski — DISMISSED** [review]
LGTM 👍

**Pull request merged**